### PR TITLE
test fixes

### PR DESCRIPTION
**Pull request by bisgaard-itis**
# Description

Fix osparc tests https://autotest.bioeng.auckland.ac.nz/jenkins/blue/organizations/jenkins/SPARC-API/detail/SPARC-API/898/pipeline. The failing tests are caused by an internal change in osparc which means that the reported state in when starting a simulation on osparc is either `PUBLISHED` or `PENDING`.

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by bisgaard-itis** [comment]
@mguidon @pcrespov

**Review by pcrespov — COMMENTED** [review]
@agarny Could you please have a look at these changes?

File: tests/test_osparc.py [hunk 1]
```diff
@@ -132,13 +132,17 @@ def test_osparc_successful_simulation(client):
     }
     r = client.post("/start_simulation", json=data)
     assert r.status_code == 200
-    check_simulation_data = json.loads(r.data)["data"]
+    json_data: dict = json.loads(r.data)
```
Comment (agarny) [inline tests/test_osparc.py:135]: For consistency, I would remove the `: dict` (same for line 173) since it's not used on line 142 (line 180, resp.).

**Review by agarny — CHANGES_REQUESTED** [review]
Looks fine to me. Just one minor request.

File: tests/test_osparc.py (hunk 1 above)
Comment (bisgaard-itis) [reply tests/test_osparc.py]: Yep, makes sense. I have fixed this.

Comment (bisgaard-itis) [reply tests/test_osparc.py]: Can I get you to merge @agarny? It seems I don't have the right permissions to do that 🤔

**Review by agarny — APPROVED** [review]

File: tests/test_osparc.py (hunk 1 above)
Comment (agarny) [reply tests/test_osparc.py]: Thanks @bisgaard-itis. It looks like I am not authorised either. @alan-wu, could you please check and merge this PR?

**Review by hsorby — APPROVED** [review]

**Comment by agarny** [comment]
Thanks @hsorby!

**Pull request merged**